### ISSUE TITLE
Add config options for setBreaksEnabled and setMarkupEscaped

### ISSUE THREAD

**Issue by ngyikp** [andreasindal/laravel-markdown#8]
There seems to be no way to toggle `setBreaksEnabled` and `setMarkupEscaped` from configuration

https://github.com/erusev/parsedown/wiki/Tutorial:-Get-Started

**Comment by ahmadyousefdev** [comment]
+1 for this , Amazing Package ! 😃

**Comment by ngyikp** [comment]
FYI, this can be accomplished with a workaround: (I put this on my AppServiceProvider)

``` php
$this->app->make('Indal\Markdown\Parser')->parsedown->setMarkupEscaped(true)->setBreaksEnabled(true);
```

**Comment by alsterholm** [comment]
Hi!

Good catch :) I'll look into it! If someone wants to make a pull request, feel free! Otherwise I'll add a patch this weekend.

**Comment by alsterholm** [comment]
Fixed in #9
